.github/actions/build-cdn/action.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Install, build & cache'
+description: 'Installs dependencies, builds all packages, and saves the cache of Nx'
+runs:
+  using: composite
+  steps:
+    - uses: ./.github/actions/setup
+      with:
+        load-cache: 'false'
+        cache-suffix: 'cdn'
+    - uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4
+      id: nx-cache
+      with:
+        path: .nx/cache
+        key: nx-${{ github.sha }}-cdn
+    - if: steps.nx-cache.outputs.cache-hit != 'true'
+      run: npm run build
+      shell: bash
```

Comment:
This is the exact same action as /build/action.yml except for the `cache-suffix: 'cdn'`. Why can't you use the build one ?